src/app/services/dayjs/index.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/no-named-as-default-member */
```

Comment:
eslint disable comment could be removed, if you change code like that:

```tsx
import dayjs, { extend } from 'dayjs';
import isBetween from 'dayjs/plugin/isBetween';

extend(isBetween);

export const dayJS = dayjs;
```

Also, why do you re-export `dayjs`, with changed name?

Reply:
Done